ess/src/API/EMBC.Tests.Integration.ESS/Submissions/SubmissionsTests.cs

```diff
@@ -364,6 +364,18 @@ public async Task CanSearchEvacuationFilesByRegistrantUserName()
             files.ShouldAllBe(f => f.PrimaryRegistrantId == registrant.Id);
         }
 
+        [Fact(Skip = RequiresDynamics)]
+        public async Task CanSearchEvacuationFilesByLinkedRegistrantId()
+        {
+            var registrant = (await GetRegistrantByUserId("CHRIS-TEST"));
+            var statuses = new[] { EvacuationFileStatus.Active, EvacuationFileStatus.Archived, EvacuationFileStatus.Completed, EvacuationFileStatus.Expired, EvacuationFileStatus.Pending };
+
+            var files = (await manager.Handle(new EvacuationFilesQuery { LinkedRegistrantId = registrant.Id, IncludeFilesInStatuses = statuses })).Items;
+
+            files.ShouldNotBeEmpty();
+            //files.ShouldAllBe(f => f.PrimaryRegistrantId == registrant.Id);
```

Comment:
can you add a check that there's 1 member in each file that has that registrant id?